patches/crosstool-NG/1.22.1/series

```diff
@@ -0,0 +1 @@
+Use-awk-hardlink.patch
```

Comment:
if you take https://github.com/crosstool-ng/crosstool-ng/pull/410/commits for crosstool-ng, then this patch is not needed.